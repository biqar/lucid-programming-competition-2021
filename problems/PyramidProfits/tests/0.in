2
Ian Jude 50
Kathy Ian 10
2
Kathy 1000
Ian 2500
