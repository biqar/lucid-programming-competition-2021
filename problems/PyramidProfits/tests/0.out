Ian 1700
Jude 1750
Kathy 50
